src/request-params.ts

```diff
@@ -24,12 +22,15 @@ type PaginationQueryParams = {
   next?: string;
 };
 
+export const extractTermParam = (term: string) =>
```

Comment:
"term" is pretty vague naming here. What about something more specific like `extractHashorOrdinalParam`?